### PR TITLE
[T2A2][T16-A4] Andrew Sugiarto

### PR DESCRIPTION
**Pull request by Azraelos**
Refactor removePrefixSign

### CODE REVIEW

**Comment by Azraelos** [comment]
Ready for review.

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -989,12 +989,12 @@ private static String extractPhoneFromPersonString(String encoded) {
 
         // phone is last arg, target is from prefix to end of string
         if (indexOfPhonePrefix > indexOfEmailPrefix) {
-            return removePrefixSign(encoded.substring(indexOfPhonePrefix, encoded.length()).trim(),
+            return removePrefix(encoded.substring(indexOfPhonePrefix, encoded.length()).trim(),
```
Comment (sooyj) [inline src/seedu/addressbook/AddressBook.java:992]: Good change: ambiguous term "sign" is removed.

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -1144,14 +1144,14 @@ private static String getUsageInfoForExitCommand() {
...
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix from the given fullString if prefix occurs at the start of the string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
+     * @param fullString  Parameter as a string
+     * @param prefix  Parameter prefix to be removed
```
Comment (sooyj) [inline src/seedu/addressbook/AddressBook.java:1150]: Correct! Comments should be updated accordingly.

File: src/seedu/addressbook/AddressBook.java [hunk 3]
```diff
@@ -1144,14 +1144,14 @@ private static String getUsageInfoForExitCommand() {
...
+     * @return  string without the prefix
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+        return fullString.replaceFirst(prefix, "");
```
Comment (sooyj) [inline src/seedu/addressbook/AddressBook.java:1154]: The code logic is now correct, good work!

**Review by sooyj — COMMENTED** [review]
Good refactoring attempt on one method.

Can try to hunt down a few example of coding convention violation for practice.

**Pull request closed without merging**